docs/quickguide.md

```diff
@@ -224,9 +185,6 @@ After that, register a new command for the device taking into account:
 
 ![](media/image03.png)
```

Comment:
quickguide.md should be removed in further modifications. It has not been removed now in order to avoid conflict when merging to master